src/components/policy/src/cache_manager.cc

```diff
@@ -755,8 +757,7 @@ void CacheManager::CheckSnapshotInitialization() {
   rpc::Optional<policy_table::UsageAndErrorCounts>& usage_and_error_counts =
       snapshot_->policy_table.usage_and_error_counts;
 
-  if (usage_and_error_counts->is_initialized() &&
-      usage_and_error_counts->app_level->is_initialized()) {
```

Comment:
@AGaliuzov No. This field is defined as `Optional<AppLevels> app_level` in the `UsageAndErrorCounts` which defines the field `Optional<UsageAndErrorCounts> usage_and_error_counts`. Thus the produced instance is a valid object.